OMEdit/OMEdit/OMEditGUI/Modeling/ModelWidgetContainer.cpp

```diff
@@ -3067,12 +3113,7 @@ void GraphicsView::keyReleaseEvent(QKeyEvent *event)
 void GraphicsView::contextMenuEvent(QContextMenuEvent *event)
 {
   /* If we are creating the connection OR creating any shape then don't show context menu */
-  if (isCreatingConnection() ||
-      isCreatingLineShape() ||
-      isCreatingPolygonShape() ||
-      isCreatingRectangleShape() ||
-      isCreatingEllipseShape() ||
-      isCreatingTextShape()) {
+  if (isCreatingShape()) {
```

Comment:
You need to add `isCreatingConnection()` back once you remove it from `isCreatingShape()`.

Reply:
Roger roger